projects/ng2-date-picker/src/lib/date-picker/date-picker.component.ts

```diff
@@ -384,6 +392,16 @@ export class DatePickerComponent implements OnChanges,
     });
   }
 
+  rangeSelected(range: IDateRange, ignoreClose?: boolean) {
+    if (!ignoreClose) {
+      this.onDateClick();
+    }
+
+    this.inputElementValue = (this.utilsService.convertToString(range.from, this.componentConfig.format))
+      + ' <> ' + (this.utilsService.convertToString(range.to, this.componentConfig.format));
```

Comment:
1. Please use template strings
2. Let's make the delimiter to be configured

Reply:
1. Ok
2. I'll work for limiters tomorrow.